src/style/App.css

```diff
@@ -0,0 +1,3 @@
+.App {
```

Comment:
The general layout matches the given wireframes in the [project requirements](https://github.com/microverseinc/curriculum-react-redux/blob/main/math-magicians/project_full_website.md) 🙌